opensearch-observability/build.gradle

```diff
@@ -247,6 +257,147 @@ testClusters.integTest {
     setting 'path.repo', repo.absolutePath
 }
 
+
+String bwcVersion = "1.1.0-SNAPSHOT"
+String baseName = "obsBwcCluster"
+String bwcFilePath = "src/test/kotlin/org/opensearch/observability/resources/bwc/"
+
+2.times {i ->
+    testClusters {
+        "${baseName}$i" {
+            testDistribution = "ARCHIVE"
+            versions = ["1.1.0","1.2.0-SNAPSHOT"]
+            numberOfNodes = 3
+            plugin(provider(new Callable<RegularFile>(){
+                @Override
+                RegularFile call() throws Exception {
+                    return new RegularFile() {
+                        @Override
+                        File getAsFile() {
+                            return fileTree(bwcFilePath + bwcVersion).getSingleFile()
+                        }
+                    }
+                }
+            }))
+            setting 'path.repo', "${buildDir}/cluster/shared/repo/${baseName}"
+            setting 'http.content_type.required', 'true'
+        }
+    }
+}
+
+List<Provider<RegularFile>> plugins = []
+
+// Ensure the artifact for the current project version is available to be used for the bwc tests
+task prepareBwcTests {
+    dependsOn bundle
+    doLast {
+        plugins = [
+                project.getObjects().fileProperty().value(bundle.getArchiveFile())
```

Comment:
could you take a look at this? https://github.com/opensearch-project/trace-analytics/pull/310#pullrequestreview-828353906 other PRs don't include the artifact for 1.2.0.0-SNAPSHOT, try removing it to see if it works?

Reply:
Yeah okay!